### PR TITLE
Bartender drinks content

### PR DESCRIPTION
**Pull request by ZeroDayDaemon**
## About the PR 
Adds moonshine drinks sprites
Sprites courtesy of @Mith-randalf.

And now addressing the elephant in the room with this PR:

Lean and Leanshine.

Lean was removed due to its ability to poison and other wacky effects, as well as it being "annoying".
I feel that its complete removal was entirely not needed, and a rework or removal of the ability to craft it would have had the exact same intended outcome.
so that's exactly what I've done, I removed the effects, and make base lean uncraftable for the time being, until the time is right when it can be re-added as a drink that can be crafted.

Having healthy discussion is welcome on this PR, what is not welcome is closing it without reasonable community and maintainer discussion.

I have Re-Added Lean and Leanshine and given them sprites.

When #6961 is completed I also plan to PR 3 bottles of moonshine and 1 leanshine as emag inventory items for the bartenders drink bottle machine

**Screenshots**

**Changelog**

:cl: Daemon
- add: Added Moonshine drink sprites
- add: Re-Added fun!

### CODE REVIEW

**Comment by moonheart08** [comment]
https://github.com/tgstation/tgstation/pull/65464

y'know i think i agree with /tg/ for once

**Comment by moonheart08** [comment]
For me to really be OK with lean being readded it needs renamed and recolored as to not be the meme of the month.

**Comment by Snowni** [comment]
I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

**Comment by ZeroDayDaemon** [comment]
> For me to really be OK with lean being readded it needs renamed and recolored as to not be the meme of the month, so we don't get people going crazy over haha purple

It can't be crafted. This is no longer an issue. Also renaming it makes no sense, since lean is an actual real world thing.

**Comment by ZeroDayDaemon** [comment]
> I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

It was in the game since 2020 with no issue, also, it has no effects in this PR, so, yeah.

**Comment by Leander-0** [comment]
> I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.

if you want to remove lean just because its a meme then you should remove Skub too.

i really dont see why all the hate to lean all of a sudden.

**Comment by ZeroDayDaemon** [comment]
> https://github.com/tgstation/tgstation/pull/65464
> 
> y'know i think i agree with /tg/ for once

For once I agree with swept on that PR.

**Comment by moonheart08** [comment]
> > I'd rather keep lean out, shit meme, and it's borderline RDM when the bartenders stock their shelves with it and people drink it. Not to mention all the people forcefeeding it to others.
> 
> if you want to remove lean just because its a meme then you should remove Skub too.
> 
> i really dont see why all the hate to lean all of a sudden.

Because it just suddenly blew up and caused our game staff a **fuckton** of unnecessary work. They otherwise have no way of combatting it as it's an LRP server, so even if half the crew is dead of severe drug poisoning from consuming lean repeatedly they just have to live with the load. The easiest/sanest route to handling something being super painful to work with like this is just to remove it. 

Also, if it has _no_ effects, then it's just not eligible to be in the game, as it's an unattainable, useless chem.

**Comment by ZeroDayDaemon** [comment]
Lean has been in the game for almost two years. And only now do people have an issue with it, because I will admit I fucked up by making it a poison. But, It is not a poison in this PR, nor can it be crafted.

**Comment by Mith-randalf** [comment]
idgaf about the lean but if this is unmerged 24 hours from now I'll resprite the moonshine jug, looking at it, it's too small (but if it does get merged its fine cos it really doesnt matter)

**Comment by ZeroDayDaemon** [comment]
So I believe I've addressed every issue and concern with it in this PR, except one:

"I don't like it, it's an unfunny meme"
That's your opinion, if we removed everything from the game that one person didn't find funny or fun, there would be no game.

**Comment by Leander-0** [comment]
> Because it just suddenly blew up and caused our game staff a **fuckton** of unnecessary work. They otherwise have no way of combatting it as it's an LRP server, so even if half the crew is dead of severe drug poisoning from consuming lean repeatedly they just have to live with the load. The easiest/sanest route to handling something being super painful to work with like this is just to remove it.
> 
> Also, if it has _no_ effects, then it's just not eligible to be in the game, as it's an unattainable, useless chem and we no longer allow that in the codebase.

drugs

When a player consumes drugs be it Meth, a blunt or lean they all know the effects and the risks of taking of taking such drugs.

THC and and smoking have no effects right now and yet the players smoke cigars and make blunts.

what will happen when other drugs like cocaine get added into the game then?

**Comment by ZeroDayDaemon** [comment]
I've reworked your concerns with lean, It now has a use and can be crafted, has been moved to soda since thats the primary filler ingredient, and cures space cold

**Comment by ZeroDayDaemon** [comment]
crafting it is more difficult and now requires access to a chemistry dispenser

**Comment by Fishfish458** [comment]
Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor.  in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.

I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.

File: Resources/Prototypes/Reagents/medicine.yml [hunk 1]
```diff
@@ -77,6 +77,21 @@
           groups:
             Brute: -2
 
+- type: reagent
+  id: Codeine
+  name: codeine
```
Comment (metalgearsloth) [inline Resources/Prototypes/Reagents/medicine.yml:82]: Do we have any other references to actual drugs? If so they should be culled.

**Review by metalgearsloth — COMMENTED** [review]
Putting in a real world poison, lean, seems like a bad idea.

File: Resources/Prototypes/Reagents/medicine.yml (hunk 1 above)
Comment (ZeroDayDaemon) [reply Resources/Prototypes/Reagents/medicine.yml:82]: Meth, Marijuana, Alcohol, Space Drugs, uhhhh, alot more i think

Comment (ZeroDayDaemon) [reply Resources/Prototypes/Reagents/medicine.yml:82]: Nicotine

Comment (ZeroDayDaemon) [reply Resources/Prototypes/Reagents/medicine.yml:82]: Codeine is usually the main active ingredient in prescription cough medicine

**Comment by ZeroDayDaemon** [comment]
> Putting in a real world poison, lean, seems like a bad idea.

What on your thoughts on why that is?

**Comment by moonheart08** [comment]
> Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> 
> I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.

References to drugs are fine

the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)

**Comment by ZeroDayDaemon** [comment]
> > Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> > I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.
> 
> References to drugs are fine
> 
> the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)

I've basically given up on lean at this point, im more confused about sloths aversion to codeine, when theres already a load of dangerous drugs in game, real ones, like alcohol and meth and nicotine

**Comment by ZeroDayDaemon** [comment]
> > > Guess I'll throw in my comment here too, but I don't think the "It has no purpose/ is dangerous to the crew" argument really has grounding in a game that adds massive varieties of chemicals/foods/drinks/items purely for additional content and RP flavor. in SS13 are drinks that can poison or burn you, foods that immediately kill your character or turn them into a borg, chemicals that can make you mind slaved to someone else.
> > > I agree that the Lean meme is pretty stupid and if references to drugs truly need to be toned down, sure, rename it or whatever, though there are tons more dangerous food/drinks/chemicals that still should be added, not that they might be unique to their interactions but because they offer variety and something to roleplay with.
> > 
> > 
> > References to drugs are fine
> > the problem is lean is a massive meme right now. I can demonstrate why this is a problem at any time if needed (by temporarily readding it to the game using admin tools. Half the crew will be dead in 10 minutes.)
> 
> I've basically given up on lean at this point, im more confused about sloths aversion to codeine, when theres already a load of dangerous drugs in game, real ones, like alcohol and meth and nicotine

also, only half the crew will be dead if lean is a poison again, which, in this PR, it is not

**Comment by ZeroDayDaemon** [comment]
I'm going to be expanding and reworking this PR since i have the time today

File: Resources/Prototypes/Reagents/medicine.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Reagents/medicine.yml:82]: All of those are restricted to varying degrees so it's a bit weird.

**Comment by ZeroDayDaemon** [comment]
Closing this, maybe another time

**Pull request closed without merging**